modules/rosa-cluster-autoscaler-ui-settings.adoc

```diff
@@ -16,25 +18,45 @@ The tables explain all the configurable UI settings when using cluster autoscali
 |Type or Range
 |Default
 
-|`balanceSimilarNodeGroups`
+|`balance-similar-node-groups`
 |If `true`, this setting automatically identifies node groups with the same instance type and the same set of labels and tries to keep the respective sizes of those node groups balanced.
 |`boolean`
 |false
 
-|`logVerbosity`
-|Sets the autoscaler log level. Default value is 1, level 4 is recommended for debugging and level 6 enables almost everything.
-|`integer`
-|1
+|`skip-nodes-with-local-storage`
+|If `true`, the cluster autoscaler never deletes nodes with pods with local storage, e.g. EmptyDir or HostPath.
+|`boolean`
+|true
 
-|`maxPodGracePeriod`
+|`max-pod-grace-period (max-graceful-termination-sec)`
 |Gives pods graceful termination time before scaling down.
 |`integer`
 |600
 
-|`skipNodesWithLocalStorage`
-|If `true`, the cluster autoscaler never deletes nodes with pods with local storage, e.g. EmptyDir or HostPath.
+|`max-node-provision-time`
+|Maximum time the cluster autoscaler waits for nodes to be provisioned.
+|`string`
+|15m
+
+|`pod-priority-threshold`
+|To allow users to schedule "best-effort" pods, which shouldn't trigger cluster autoscaler actions. They only run when there are spare resources available.
+|`integer`
+|-10
+
+|ignore-daemonsets-utilization`
```

Comment:
I can see a format difference in  https://63942--docspreview.netlify.app/openshift-rosa/latest/rosa_cluster_admin/rosa-cluster-autoscaling#rosa-cluster-autoscale-cli-interactive_rosa-cluster-autoscaling
(`ignore-daemonsets-utilization` has a different format and ends with a "tickmark")

![format-diff](https://github.com/openshift/openshift-docs/assets/829045/ef9e810a-7ebe-4580-a20a-067eb8b5dfe7)